en: update docs in get started (#15)
pingcap/docs-tidb-operator: en/deploy-tidb-from-kubernetes-gke.md

If you see `Ready` for all nodes, congratulations! You've setup your first Kubernetes cluster.

## Deploy TiDB Operator and TiDB cluster

1. Install Helm and add the Helm chart repository maintained by PingCAP. For details, refer to [Use Helm](tidb-toolkit.md#use-helm)

> 2. Deploy TiDB Operator. For details, refer to [Install TiDB Operator](deploy-tidb-operator.md#install-tidb-operator).

Reviewer: ```suggestion
2. [Deploy TiDB Operator](deploy-tidb-operator.md#install-tidb-operator).
```

select tidb_version();
```

> If you did not specify a password in the process of installation, set one now:

Reviewer: ```suggestion
If you did not specify a password during installation, set one now:
```
